blocks/blog-home/blog-home.js

```diff
@@ -5,6 +5,115 @@ import {
 const NUM_CARDS_SHOWN_AT_A_TIME = 6;
 let loadMoreElement;
 const MODE = 'blog-home';
+let url = new URL(window.location.href);
+const params = new URLSearchParams(url.search);
+export const selectedAudience = params.get('audience') ? params.get('audience').split(',') : [];
+export const selectedTopics = params.get('topics') ? params.get('topics').split(',') : [];
+export const selectedContentTypes = params.get('content-types') ? params.get('content-types').split(',') : [];
+
+function getSetParam() {
+  if (selectedAudience.length > 0) {
+    params.set('audience', selectedAudience);
```

Comment:
Nitpicking: I wonder if there's a better way to pull from the string value of each filter instead of explicitly setting the string here. For instance, you can grab the value for each filter by grabbing the value from the `.list-title` selector (see attached). This way if the label of each filter ever gets updated it changes the URL parameter value as well. 

![Screenshot 2023-07-24 at 2 41 16 PM](https://github.com/hlxsites/merative2/assets/1815714/d89a9168-8328-4c6b-85bc-01b64fd28758)